db/migrate/20230403150658_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+    rename_column :comments, :user_id, :author_id
+  end
```

Comment:
- [ ] Same here,  To show that, the  `'author_id'` reference the User class. Kindly write the query like this: 
```ruby
   add_reference :comments, :author, null: false, foreign_key: { to_table: :users }
```
- This will create author_id and a corresponding index